starter/source/elements/reader/hm_read_merge_node.F

```diff
@@ -84,9 +84,9 @@ SUBROUTINE HM_READ_MERGE_NODE(X,LSUBMODEL,UNITAB,IGRNOD,MERGE_NODE_TAB,
 C
       TOL_DEF = ZERO
       FLAG_FULL_MERGE = 0
-      ALLOCATE(TAGNOD1(NUMNOD),TAGNOD2(NUMNOD))
-      TAGNOD1 = 0
-      TAGNOD2 = 0
+      ALLOCATE(TAGNOD1(NUMNOD),TAGNOD2(NUMNOD),TAGNOD_TEMP(NUMNOD))
```

Comment:
Hi @bertrand-altair 

you misses the de-allocation of TAGNOD_TEMP array, I let you correct this please.

Thanks,
Sébastien